client/my-sites/plans/current-plan/header.jsx

```diff
@@ -137,6 +142,8 @@ class CurrentPlanHeader extends Component {
 						<HappinessSupport
 							isJetpack={ !! selectedSite.jetpack && ! isAutomatedTransfer }
 							isPlaceholder={ isPlaceholder }
+							showLiveChatButton={ this.isEligibleForLiveChat() && ! isAutomatedTransfer }
```

Comment:
Now that I'm looking at it again, I dont' think this extra check makes any sense. AT sites won't have a Jetpack business plan ever.